source/usage-examples/deleteOne.txt

```diff
@@ -23,7 +23,7 @@ matched:
    :language: go
    :dedent:
 
-View a `fully runnable example <{+example+}/deleteOne.go>`__
+View a `fully runnable example. <{+example+}/deleteOne.go>`__.
```

Comment:
I only see `delete.go` in this PR. Are `deleteOne.go` and `deleteMany.go` kept elsewhere?

Reply:
So the update here was to add a period at the end of the sentence since it was missing. I'm seeing here in this instance, I added two, so I'll update that in my revisions.